distributed/shuffle/_worker_extension.py

```diff
@@ -696,10 +703,10 @@ async def _get_shuffle_run(
                 shuffle_id=shuffle_id,
             )
         if run_id < shuffle.run_id:
```

Comment:
Minor: use the same ordering for the conditions (`shuffle.run_id > run_id`) as in `_restrict_task` in the scheduler extensions?

Reply:
Done